src/System.Management.Automation/engine/remoting/server/ServerPowerShellDriver.cs

```diff
@@ -21,11 +21,11 @@ internal class ServerPowerShellDriver
         private bool _extraPowerShellAlreadyScheduled;
         private PowerShell _extraPowerShell;     // extra PowerShell at the server to be run after localPowerShell
         private PSDataCollection<PSObject> _localPowerShellOutput; // output buffer for the local PowerShell
-        // that is associated with this
-                                                 // powershell driver
-        // associated with this powershell
-        // data structure handler object to handle all
-        // communications with the client
+                                                                   // that is associated with this
```

Comment:
should probably reformat this comment altogether to just a few lines

Reply:
All comments was reformatted.